buildSrc/src/main/groovy/openhouse.maven-publish.gradle

```diff
@@ -82,7 +76,6 @@ publishing {
 }
 
 artifacts {
-  archives file: file('$pomPath'), name: '$name', type: 'pom'
```

Comment:
I guess removal of explicitly passing helps with gradle 7 upgrade as well.

Reply:
I see. interesting! ty.